docs/contributor/telemetry-load-test/assets/log-fluentbit-test-setup.yaml

```diff
@@ -84,7 +84,7 @@ spec:
       containers:
         - args:
             - --config=/etc/collector/config.yaml
-          image: europe-docker.pkg.dev/kyma-project/prod/tpi/otel-collector:0.93.0-fdc9e6c4
```

Comment:
Was this intentional?

Reply:
yes, we will use official OTEL image release for mock backend, logging mock backend had different image than trace and metric mock backend

Reply:
But shouldn't we also use 0.94.0?

Reply:
And what is missing in our image? We add the fluentforwardreceiver [here](https://github.com/kyma-project/third-party-images/blob/main/otel-collector/builder-config.yaml#L16)